grails-app/controllers/org/pih/warehouse/picklist/ImportPickCommand.groovy

```diff
@@ -25,6 +25,27 @@ class ImportPickCommand implements Validateable {
         })
         expirationDate(nullable: true)
         binLocation(nullable: true)
-        quantity(nullable: false, min: 0)
+        quantity(nullable: false, validator: { val, obj ->
```

Comment:
I see what you mean now regarding this comment.

> Now if we set quantity as a type Integer, we lose the ability to even get to the custom validators because it will throw an exception something along the lines of Can't cast String to an Integer in constructor when assigning this.quantity = quantity, which is why I changed the type of the quantity so we can assign the quantity and then our validator can validate the value.

Reply:
So i think what you have is the right approach for now. 

We could/should explore if there's a way to do some of the parsing / validation logic in the CSVMapReader to avoid these kinds of workarounds. Or use a different library. 

But that's out of scope for this ticket.

Reply:
This definitely sounds like something we should discuss in the design of importers ticket (OBID idea garden ticket)